exercises/06_repetition/README.md

```diff
@@ -38,7 +38,9 @@ macro_rules! listing_literals {
 
 fn main() {
     let vec: Vec<&str> = listing_literals!(the "lion" and the "witch" and the "wardrobe");
+    assert_eq!(vec, vec!["lion", "witch", "wardrobe"]);
```

Comment:
No assertions in main pls. See chapter 2 for rationale. Happy to do this, but it needs to move out to a function.

Reply:
This was a small brain moment from me -- macros are fine in readme.